libvast/src/system/exporter.cpp

```diff
@@ -47,6 +47,7 @@ void ship_results(stateful_actor<exporter_state>* self) {
 }
 
 void shutdown(stateful_actor<exporter_state>* self) {
+  // TODO: enable clean shutdown on CTRL + C ...?
```

Comment:
CTRL+C elicits an exit message from the `vast` binary to the NODE, which in turn shuts down all components. This includes the TRACKER, which [shuts down the EXPORTER](https://github.com/vast-io/vast/blob/b19541472566b16f3418c933050ced86c7a48ccb/libvast/src/system/tracker.cpp#L65). So for proper handling of CTRL+C, the EXPORT must only handle `exit_reason::user_shutdown` properly.